profiles/kentik_snmp/arista/arista.yml

```diff
@@ -65,3 +69,34 @@ metrics:
         column:
           OID: 1.3.6.1.4.1.30065.3.6.1.2.1.2
           name: aristaEgressQueueIndex
+  # Memory Utilization
+  - MIB: HOST-RESOURCES-MIB
+    table:
+      OID: 1.3.6.1.2.1.25.2.3
+      name: hrStorageTable
+    symbols:
+      - OID: 1.3.6.1.2.1.25.2.3.1.5
+        name: hrStorageSize
+        tag: MemoryTotal
```

Comment:
I have a nagging concern about tagging everything in hrstorage as memory.
I'll have to look around to see how its being used currently and what the impact would be of messing with them but just wanted to raise the point.